instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/utils/coroutines/Adapters.kt

```diff
@@ -77,6 +81,22 @@ suspend fun MapboxNavigation.requestRoutes(options: RouteOptions) =
         }
     }
 
+suspend fun MapboxNavigation.setNavigationRoutesSync(
```

Comment:
Usually people call `sync` the one which blocks current thread. Maybe 
```suggestion
suspend fun MapboxNavigation.setNavigationRoutesAsync(
```

or even
```suggestion
suspend fun MapboxNavigation.setNavigationRoutes(
```
?